assets/src/components/loggedInAs.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+export const LoggedInAs = ({ email }: { email: string }) => (
+  <div className="c-logged-in-as__container">
+    <div className="c-logged-in-as__icon">{email[0]?.toUpperCase() ?? ""}</div>
```

Comment:
This hacks in the username icon; whenever that's ready, we should use that here instead.

Reply:
No more! https://github.com/mbta/skate/pull/2383/commits/8fb447e8adc48f1158038538c65f40796cb520eb